### PR TITLE
Do not interrupt redact sequences because of exceptions on backfilling

### PR DESCRIPTION
**Pull request by maranda**
Self explanative, do not interrupt the loop and just carry on with the remaining entries in the queue.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -86,16 +86,20 @@ export async function redactUserMessagesIn(client: MatrixSendClient, managementR
...
+                        await client.redactEvent(targetRoomId, victimEvent['event_id']);
+                    } else {
+                        await managementRoom.logMessage(LogLevel.WARN, "utils#redactUserMessagesIn", `Tried to redact ${victimEvent['event_id']} in ${targetRoomId} but Mjolnir is running in no-op mode`, targetRoomId);
+                    }
                 }
-            }
-        });
+            });
+        } catch (error) {
+            await managementRoom.logMessage(LogLevel.DEBUG, "utils#redactUserMessagesIn", `Caught an error while trying to redact messages for ${userIdOrGlob} in ${targetRoomId}: ${error}`, targetRoomId);
```
Comment (Gnuxie) [inline src/utils.ts:101]: The `LogLevel` should probably be `ERROR`, this still isn't normal and we're going to want to know about it.

Comment (maranda) [reply src/utils.ts]: Amended in 6fc00b25684e724f18781fd43f519dd85c69dbe2

**Comment by H-Shay** [comment]
@turt2live - any issues with merging this (modulo the use of victim, which I can fix)? LGTM

**Review by turt2live — APPROVED** [review]
Thanks! This looks good to me as a thing.

We might need to sanitize `error` a bit, but better to see it used in practice I think.

**Pull request merged**